core/src/consensus/stake/actions.rs

```diff
@@ -77,7 +101,56 @@ impl Decodable for Action {
                     quantity: rlp.val_at(2)?,
                 })
             }
+            ACTION_TAG_CHANGE_PARAMS => {
+                let item_count = rlp.item_count()?;
+                if item_count < 4 {
+                    return Err(DecoderError::RlpIncorrectListLen {
+                        expected: 4,
+                        got: item_count,
+                    })
+                }
+                let metadata_seq = rlp.val_at(1)?;
+                let params = rlp.val_at(2)?;
+                let signatures = (3..item_count).map(|i| rlp.val_at(i)).collect::<Result<_, _>>()?;
```

Comment:
`.map(rlp.val_at)` without the closure doesn't work?

Reply:
It doesn't work.

Reply:
Actually, I cannot understand what you said. How could you remove the closure here?
`map` takes a function that has 1 argument, and `val_at` is a method that takes 1 argument. It means `val_at` needs 2 parameters.

Reply:
Okay, I understood. Thank you.